src/vs/workbench/services/hover/browser/hoverService.ts

```diff
@@ -61,6 +61,20 @@ export class HoverService implements IHoverService {
 		const provider = this._contextViewService as IContextViewProvider;
 		provider.showContextView(new HoverContextViewDelegate(hover, focus));
 		hover.onRequestLayout(() => provider.layout());
+		if (options.forwardClickEvent) {
+			hoverDisposables.add(addDisposableListener(hover.domNode, EventType.CLICK, e => {
+				// Forward the click to the target element(s)
+				if (options.target) {
+					if ('targetElements' in options.target) {
+						for (const element of options.target.targetElements) {
+							element.dispatchEvent(new MouseEvent(e.type, { ...e, bubbles: true }));
+						}
+					} else {
+						options.target.dispatchEvent(new MouseEvent(e.type, { ...e, bubbles: true }));
+					}
+				}
```

Comment:
Isn't it bad to click on all target elements as it could easily cause multiple click handlers to go off? Should `forwardClickEvent` be a callback or an element instead?

Reply:
Agreed, switched to a callback.